src/AppProviders.tsx

```diff
@@ -64,13 +58,12 @@ export function AppProviders({ children }: ProvidersProps) {
             <PluginProvider>
               <ColorModeScript storageKey='ss-theme' />
               <ChatwootWidget />
-              <ChakraProvider theme={theme} colorModeManager={manager} cssVarsRoot='body'>
-                <ToastContainer />
```

Comment:
This was causing double toasts since years, killing custom styles as it's not able to use the chakra theme

Reply:
the fuark 
![pikachusurprise](https://media4.giphy.com/media/v1.Y2lkPTc5MGI3NjExeXgzbnViNDg0czdmd2syYW42Z3plYnVlbHhmN3J5OGNjdXh6ZW5kZyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/6nWhy3ulBL7GSCvKw6/giphy.gif)